lib/OpenQA/Worker/Engines/isotovideo.pm

```diff
@@ -242,8 +242,8 @@ sub engine_workit {
             sleep 5 and update_setup_status until $rsync_request->processed;
             my $exit = $rsync_request->result;
 
-            log_info("rsync: " . $rsync_request->output, channels => 'autoinst');
-            return {error => "Failed to rsync tests: exit " . $exit} unless $exit == 0;
+            log_info("rsync: " . $rsync_request->output, channels => 'autoinst') if $rsync_request->output;
```

Comment:
Method calls are really expensive in Perl. While avoiding a duplicate method call might not matter much here it's a good habit to pick up. And you can use interpolation instead of concatenation.
```perl
if (my $output = $rsync_request->output) { log_info("rsync: $output", channels => 'autoinst') }
```

Reply:
good hint, thx